spec/faraday/options/env_spec.rb

```diff
@@ -27,6 +27,12 @@
     expect(ssl.fetch(:verify, true)).to be_falsey
   end
 
+  it 'handle verify_hostname when fetching' do
+    ssl = Faraday::SSLOptions.new
+    ssl.verify_hostname = true
+    expect(ssl.fetch(:verify, true)).to be_truthy
```

Comment:
This should probably be `expect(ssl.fetch(:verify_hostname, false)).to be_truthy`.
At the moment this test passes because `:verify` is not set and you default the fetch to `true`.

Reply:
Ahhh... Thanks for pointing out silly mistake.